app/controllers/short_url_requests_controller.rb

```diff
@@ -80,10 +80,10 @@ def get_short_url_request
   end
 
   def create_short_url_request_params
-    params[:short_url_request].permit(:from_path, :to_path, :reason, :organisation_slug, :confirmed)
+    params[:short_url_request].permit(:from_path, :to_path, :reason, :route_type, :organisation_slug, :confirmed)
```

Comment:
What happens with existing redirects (in the local database) that do not have a route_type when the are listed or updated. Would we need a migration?